lib/Basics/ResultT.h

```diff
@@ -172,7 +172,7 @@ class ResultT {
 
   T const& operator*() const& { return get(); }
 
-  T&& operator*() && { return get(); }
+  T&& operator*() && { return std::move(get()); }
```

Comment:
Does this really make a difference?

Reply:
The difference of compiling and not compiling, yes. `get` returns an l-value. Without `std::move` you can not return an r-value reference.